src/remediations/controller.read.js

```diff
@@ -128,6 +132,27 @@ exports.list = errors.async(async function (req, res) {
     trace.event('Fetch remediation details from DB');
     let remediations = await queries.loadDetails(req.user.tenant_org_id, req.user.username, rows);
 
+    // Check for name in fields query param:
+    // fields[data]=name
+    if (_.get(req, 'query.fields.data', []).includes('name')) {
+        trace.event('Include name data');
+        let plan_names = await queries.getPlanNames(
+            req.user.tenant_org_id
+        );
+
+        plan_names = _.map(plan_names, name => name.toJSON())
```

Comment:
I'm guessing because the sequelize query returns an array of model objects with lots of extra properties that don't go in the response.  `toJSON()` is idiomatic sequelize for extracting just the data.

Reply:
Yeah, the `toJSON` was to get it in the correct format to pass to `format.planNames(..)` but honestly it might not be necessary because I can still get the name info without doing `toJSON`. 

But yeah chaining it is a good idea if I leave it in there

Reply:
I got rid of this `toJSON` line and did all of the formatting in `planNames` in `remediations.format.js`